documentation/cookbook/working-with-existing-databases.markdown

```diff
@@ -15,23 +15,24 @@ Propel uses an abstract XML schema file to represent databases (the [schema](/do
 
 To generate a schema file, create a new directory for your project & specify the connection information in your *configuration file* for that project. For example, to create a new project, `legacyapp`, follow these steps:
 
- 1. Go to the `legacyapp` project directory anywhere on your filesystem:
+1. If your database has some 3rd party application tables, you can [add](/documentation/reference/configuration-file.html#exclude-tables) it to `exclude_tables` configuration node.
```

Comment:
s/3rd/third, also you should rather write "you can add *them* to *the* `exclude_tables` configuration node".